### PR TITLE
Update intro.rst

### PR DESCRIPTION
**Pull request by TheDragon246**
Updated introduction

### CODE REVIEW

**Comment by adswa** [comment]
There is a small conflict in this PR. :) If you need help solving it, just ping anyone :)

File: docs/source/intro.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
-Intro 
-=====
+Introduction
+------------
+-Goal of the tool: To extend git-annex special remote implementation to Open Science Framework (OSF). This allows for OSF repository to be accessible via repository hosting services, and allows for direct access to OSF stored data via a repository hosting API, for example 'git clone' in Github.
+-What can I use this tool for?: The tool is most beneficial for easy access to data stored on OSF via Github. If you are sharing your dataset via OSF and code via Github, this will allow smooth integration of both along with unified version management provided by Datalab.
```
Comment (SRSteinkamp) [inline docs/source/intro.rst:4]: Would suggest to avoid the `?:`  Maybe just delete the `:`.

Comment (TheDragon246) [reply docs/source/intro.rst]: updated. please check.

**Comment by adswa** [comment]
I've resolved the conflict by combining the existent content and will merge this in for now - more progress can be made on top of this. :)

**Comment by adswa** [comment]
I couldn't push fixes to your branch, so instead I have taken changes of yours into #46. Will close this in favor of it, but your commits aren't lost.

**Pull request closed without merging**